src/adapter/src/catalog/storage/stash.rs

```diff
@@ -74,6 +74,9 @@ pub const DEFAULT_PRIVILEGES_COLLECTION: TypedCollection<
     proto::DefaultPrivilegesKey,
     proto::DefaultPrivilegesValue,
 > = TypedCollection::new("default_privileges");
+// If you add a new collection, then don't forget to write a migration that initializes the
+// collection either with some initial values or as empty. See
+// [`mz_stash::upgrade::v17_to_v18`] as an example.
```

Comment:
It is a bit of a bummer that the initialization logic needs to be duplicated in the initialize function and in the update migration. It's not a huge priority, but there's probably a clever way of only defining it once and using it in both places.